### PR TITLE
add tests for quality estimator

### PR DESCRIPTION
**Pull request by elenlee**

### CODE REVIEW

File: tests/test_qualityestimator.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    Test estimate emotions.
+    """
+
+    @classmethod
+    def setup_class(cls):
+        super().setup_class()
+        cls.defaultDetector = cls.faceEngine.createFaceDetector(DetectorType.FACE_DET_DEFAULT)
+        cls.warper = cls.faceEngine.createWarper()
+        cls.qualityEstimator = cls.faceEngine.createWarpQualityEstimator()
+        cls.warp = [cls.warper.warp(detector.detectOne(VLImage.load(filename=ONE_FACE))) for detector in cls.detectors]
```
Comment (gluhar2006) [inline tests/test_qualityestimator.py]: warp -> warps

File: tests/schemas.py [hunk 2]
```diff
@@ -26,3 +26,16 @@
     "additionalProperties": False,
     "required": ["rect", "score"],
 }
+
+QUALITY = {
```
Comment (gluhar2006) [inline tests/schemas.py]: QUALITY -> QUALITY_SCHEMA

File: tests/test_qualityestimator.py (hunk 1 above)
Comment (elenlee) [reply tests/test_qualityestimator.py]: Готово

File: tests/schemas.py (hunk 2 above)
Comment (elenlee) [reply tests/schemas.py]: Готово

**Pull request merged**